### PR TITLE
Setting Linters

### PR DESCRIPTION
**Pull request by nejidevelops**
# Setting Linters

In this project, I was able to:
- Create a [gitignore](https://github.com/NewtonOmbese/Hello-Microverse/blob/setting-linters/.gitignore) file and include the node_modules directory and test.md file in it.
- Able to install [Stylelint](https://stylelint.io/) and [Webhint](https://webhint.io/) packages to my package json file
- Able to configure my [linters.yml](https://github.com/NewtonOmbese/Hello-Microverse/blob/setting-linters/.github/workflows/linters.yml), [.hintrc](https://github.com/NewtonOmbese/Hello-Microverse/blob/setting-linters/.hintrc) and [.stylelintrc.json](https://github.com/NewtonOmbese/Hello-Microverse/blob/setting-linters/.stylelintrc.json)

### CODE REVIEW

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @NewtonOmbese, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights
Well-structured files ✅
All linter checks are passing ✅

## Required Changes ♻️
- [ ] Kindly note that it is important to write a **descriptive pull request summary** because it helps other developers and team members to understand the changes that have been made. 
    - Additionally, writing a descriptive pull request summary help code reviewer know exactly the changes you have made, and that way they can focus on just those particular changes made. Therefore, taking the time to write a descriptive pull request summary can improve the efficiency and effectiveness of the code review process, and ultimately help ensure the overall quality of the code.
    
- [ ] Kindly consider changing the license `path` in your `README` from `LICENSE` to `MIT` since the `MIT.md` file contains the license of the project. Currently, it's not working as it should and due to that when a user clicks on the license link in your `README` they only see a `404` page. See below for more details:
Screenshot 2023-05-30 at 13 51 01


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -263,7 +263,7 @@ I would like to thank Microverse and Najeeb who we will surely finish this journ
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (Gambit142) [inline README.md]: - [OPTIONAL] When adopting Microverse's README template, please remove the Microverse logo. If you have your logo, you can use it in place of the Microverse's own but if you don't, please remove it entirely. 👍 
- [OPTIONAL] When adopting Microverse's README template, please remove the placeholder instruction for creating a LICENSE. The placeholder directions can be seen in the screenshot below:

image

**Review by Gambit142 — APPROVED** [review]
Hi @NewtonOmbese 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- It has HTML and CSS files ✔️
- It has node_module and test.md git ignored ✔️
- README.md file is professional ✔️ 

### Optional Suggestion
- _**Kindly consider the comment tagged [OPTIONAL] under this review summary**_

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (nejidevelops) [reply README.md]: Thank You I'll sure be doing that

**Pull request merged**